README.md

```diff
@@ -6,13 +6,11 @@ Audit reports (note some reports are kept private upon request from clients):
 ### 4. [Yearn veYFI by residents](https://github.com/yacademy/audits/blob/main/Yearn/yAcademy_veYFI_review.pdf)
 ### 5. [Morpho by residents](https://github.com/yacademy/audits/blob/main/Morpho/yAcademy_morpho_review.pdf)
 ### 6. [Ohm Bond by residents](https://github.com/yacademy/audits/blob/main/Olympus_DAO/yAcademy_ohm_bond_review.pdf)
+### 7. [Manifold OpenMEV by resiednts](https://github.com/yacademy/audits/blob/main/Manifold/yAcademy_OpenMEV_review.pdf)
```

Comment:
[Fix typo in PR #10 first](https://github.com/yacademy/audits/pull/10#discussion_r906218057)